test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -52,6 +124,72 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	Describe("[rfe_id: 64006][Dynamic OVS Pinning]", Ordered, func() {
+		Context("[Performance Profile applied]", func() {
+			It("[test_id:64097] Activation file is created", func() {
+				cmd := []string{"ls", activation_file}
+				out, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
+				Expect(err).ToNot(HaveOccurred(), "file %s doesn't exist ", activation_file)
+				Expect(out).To(Equal(activation_file))
+			})
+		})
+		Context("[Performance Profile Modified]", func() {
+			BeforeEach(func() {
+				initialProfile = profile.DeepCopy()
+			})
+			It("[test_id:64099] Activation file doesn't get deleted", func() {
+				performanceMCP, err := mcps.GetByProfile(profile)
+				Expect(err).ToNot(HaveOccurred())
+				policy := "best-effort"
+				// Need to make some changes to pp , causing system reboot
+				// and check if activation files is modified or deleted
+				profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
+				Expect(err).ToNot(HaveOccurred(), "Unable to fetch latest performance profile")
+				currentPolicy := profile.Spec.NUMA.TopologyPolicy
+				if *currentPolicy == "best-effort" {
+					policy = "restricted"
+				}
+				profile.Spec.NUMA = &performancev2.NUMA{
+					TopologyPolicy: &policy,
+				}
+				By("Updating the performance profile")
+				profiles.UpdateWithRetry(profile)
+				By("Applying changes in performance profile and waiting until mcp will start updating")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+				By("Waiting for MCP being updated")
+				mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+				By("Checking Activation file")
+				cmd := []string{"ls", activation_file}
```

Comment:
same here for linux commands

Reply:
as mentioned above, we need to use the ls command to check the activation file exists on worker node .